draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -931,29 +935,37 @@ REQ-SIGNAL-EXPOSURE-FAIRNESS:
 : Means to expose the signal independent of the application should be
 considered. An example of such exposure is OS APIs.
 
+REQ-NETWORK-SEEKS-LOAD-DOWN:
+: During detected reactive event, the network must implement a
+reactive traffic policy to reduce or offload some of the traffic.
+: This may involve utilizing alternative network attachments
+available to the client (e.g., Wi-Fi).
+
 REQ-CONTINUITY:
 : Handover from one radio or router to another should continue to
 provide same service level.
+: The network must deploy wireless routers closer to users to reduce link
```

Comment:
Deploying network routers closer to the user/client does not resolve "continuity" requirement. What matters is that when the session is re-established (after handover), any negotiation of metadata is handling during the session re-establishment.

Reply:
```
The general trend in wireless networks is to deploy the wireless
router closer to the user. This can help with low link latency but
can result in more frequent handovers as the user moves between
routers.
```

This was already there in the text. I just moved it under the requirement itself since it seems to be referring to a way it is done today (or at least reads so to me). Do we remove the whole statement altogether then?

Reply:
it doesn't matter where or how things are deployed; what matters is the radio interface can provide feedback to the thing-doing-rate-limiting.  Which might be itself, but it still has to provide that feedback internally. It might be a different physical device or a VM or, in 5G, a virtualized function.  Doesn't matter at our level of requirements.

Reply:
Agreed! Removed those lines altogether and stated the requirement for what it is, succinctly.

Reply:
@Sridhar: The trend to deploy wireless routers closer to the user is driven in 3GPP by capacity and latency concerns. That this results in more frequent handovers was to motivate the problem/requirement for continuity after handover. (i.e., unlike 4G-like centralized deployments, the lack of continuity for metadata would result in many users/sessions being affected). 
Just wanted to clarify: I don't see this as a big issue and what's in the draft is fine.